sal_interfaces/Scheduler/Scheduler_Events.xml

```diff
@@ -1177,4 +1177,110 @@
       <Count>1000</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>Scheduler</Subsystem>
+    <EFDB_Topic>Scheduler_logevent_observingMode</EFDB_Topic>
```

Comment:
What is this? I don't think this was in the document we wrote, no?

Reply:
This is something else that LOVE needs.. but it is related to the change we are about to do.. It will basically tell what the Scheduler is doing at the moment (e.g. executing a user provided block)